python/modules/python_connection.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/python
+import importlib.util
+import logging
+
+logger = logging.getLogger("IPIQA.python_connection")
+
+def execute_python_script(param_dict, python_script_path, function_name):
+    p_source = importlib.util.spec_from_file_location("external_module", python_script_path)
```

Comment:
Added some comments which explains how to use module. 
Yes, I load it as module, get an assigment for function with defined name and than execute it.
If scripts are bigger than one file, user should take care of proper imports i.e. import other modules by using their absolute paths. Now I'm kinda curious how does it work when it comes to r_connection.